### ISSUE TITLE
Alt text missing on some images

### ISSUE THREAD

**Issue by briggySmalls** [briggySmalls/skin-deep-server#135]
Archive images have no alt text, but _do_ have the 'alt' attribute.

It should be investigated if this is a bug or if there is just no alt text available. On the whole the text 'alt' should not be there if there is nothing to assign to it. But all images _should_ be displayed with alt text anyways, so it could be that the image title should be pulled in here.

**Comment by briggySmalls** [comment]
Looks like less of a bug and more a wordpress design. It would probably be best to use the image title if no alt text is provided, although that won't be that helpful to screen readers often and lighthouse didn't complain...

**Comment by briggySmalls** [comment]
Quite a few SEO checkers mark this as an accessibility failure. However google lighthouse doesn't, so I don't know what to believe.
